sdk/src/instructions/event.ts

```diff
@@ -74,20 +60,18 @@ export const event = (sdk: CubikSDK) => {
         sdk.programId
       );
     },
-    getEventJoin: (pda: web3.PublicKey) => {
-      return sdk.program.account.eventJoin.fetch(pda);
-    },
 
     get: (pda: web3.PublicKey) => {
       return sdk.program.account.event.fetch(pda);
     },
-    getEventJoinPDA: (
+    
+    getEventParticipantPDA: (
       eventAccount: web3.PublicKey,
       projectAccount: web3.PublicKey
     ) => {
       return web3.PublicKey.findProgramAddressSync(
         [
-          Buffer.from("event_join"),
+          Buffer.from("eventparticipant"),
```

Comment:
make a file for the constant value and use it here, same as what I did in contract